File: kits/leanplum/leanplum-7/leanplum-example/src/main/java/com/mparticle/example/leanplum/ExampleApplication.java
package com.mparticle.example.leanplum;

import android.app.Application;

import com.mparticle.MParticle;
import com.mparticle.MParticleOptions;

/**
 * Leanplum Kit Example Application
 */
public class ExampleApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        //The Leanplum kit uses Firebase by default
        //invoke this API *before* starting the mParticle SDK
        //if you'd still like to use GCM
        //LeanplumKit.disableFirebase = true;

        MParticle.start(
                MParticleOptions.builder(this)
                        .credentials("REPLACE ME", "REPLACE ME")
                        .build()
        );
    }
}

File: example/src/main/java/com/rokt/example/ExampleApplication.kt
package com.rokt.example

import android.app.Application
import com.mparticle.MParticle
import com.mparticle.MParticleOptions
import com.rokt.roktsdk.Rokt

class ExampleApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        val options: MParticleOptions = MParticleOptions.builder(this)
            .credentials(
                "MockKey",
                "MockSecret"
            )
            .environment(MParticle.Environment.Development)
            .logLevel(MParticle.LogLevel.VERBOSE)
            .build()


        MParticle.start(options)
        Rokt.init("2754655826098840951", "1.0.0",this)
    }
}